lib/dashboard.js

```diff
@@ -30,41 +30,94 @@ Dashboard.prototype.onEvent = function (event) {
   this.screen.render();
 };
 
-Dashboard.prototype._createView = function () {
-  // fixes weird scrolling issue
-  var container = blessed.box({});
-  this.screen.append(container);
+var stdOutHeight = 0.5;
+var stdOutWidth = 0.75;
+
+var metrics = [CpuView, EventLoopView, MemoryView];
+var metricsCount = metrics.length;
+
+var stdOutPosition = function (parent, interleave) {
+  return {
+    left: 0,
+    width: Math.ceil(parent.width * stdOutWidth),
+    top: 0,
+    height: interleave ? parent.height : Math.ceil(parent.height * stdOutHeight)
+  };
+};
+
+var stdErrPosition = function (parent) {
+  return {
+    left: 0,
+    width: Math.ceil(parent.width * stdOutWidth),
+    top: Math.ceil(parent.height * stdOutHeight),
+    height: "50%"
+  };
+};
 
+var metricsPosition = function (index, parent) {
+  var top = Math.ceil(index / metricsCount * parent.height);
+  var bottom = Math.ceil((index + 1) / metricsCount * parent.height);
+  return {
+    top: top,
+    height: bottom - top,
+    left: Math.ceil(parent.width * stdOutWidth),
+    width: Math.floor(parent.width * (1 - stdOutWidth))
+  };
+};
+
+Dashboard.prototype._logStream = function (streamView, data) {
+  var lines = data.replace(/\n$/, "");
```

Comment:
i think it'd be good to put this in a separate PR since it's not related to the rest of these changes

and maybe have it enabled by an option so the default behavior stays the same - @jasonwilson what do you think?